tests/templates/template_with_nonunique_slots.html

```diff
@@ -0,0 +1,4 @@
+{% load component_tags %}
+<header>{% slot "header" %}Default header{% endslot %}</header>
+<main>{% slot "header" %}Default main header{% endslot %}</main> {# <- whoops! slot name 'header' used twice.
```

Comment:
Interesting... This wasn't caught in testing because the only test that uses it performs an _assertRaises_ for a generic TemplateSyntaxError. Because the ill-formed tag raises the same error, the test reported success when in fact it wasn't testing what we're actually interested in. Good to know going forward